plugin.xml

```diff
@@ -23,6 +24,10 @@
             <clobbers target="RemotePackage" />
         </js-module>
 
+        <js-module src="bin/www/syncStatus.js" name="syncStatus">
+            <clobbers target="SyncStatus" />
```

Comment:
Is it idiomatic on the Cordova side for this to be a global enum? On the RN side, I made the SyncStatus enum hang off the CodePush module itself, since RN relies on modules instead of global vars